cachelib/cachebench/util/CacheConfig.h

```diff
@@ -41,6 +41,28 @@ class CacheMonitorFactory {
   virtual std::unique_ptr<CacheMonitor> create(Lru2QAllocator& cache) = 0;
 };
 
+struct MemoryTierConfig : public JSONConfig {
+  MemoryTierConfig() {}
+
+  explicit MemoryTierConfig(const folly::dynamic& configJson);
+  MemoryTierCacheConfig getMemoryTierCacheConfig() {
```

Comment:
Please add a docstring for this function.

Reply:
Done